omod/src/main/java/org/openmrs/module/adminui/page/controller/metadata/privileges/ManagePrivilegesPageController.java

```diff
@@ -22,5 +38,33 @@
 	public void get(PageModel model, @SpringBean("userService") UserService userService) {
 		model.addAttribute("privileges", userService.getAllPrivileges());
 	}
-	
+
+
+    public String post(PageModel model, 
+            		   @RequestParam(value = "privilegeName", required = false) @BindParams Privilege privilege,
+                       @RequestParam("action") String action,
```

Comment:
There is no other action anyways except for delete so you don't need the action parameter and the check on line 53 below